### DISCUSSION TITLE
Tilsberk Headup

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#583] [Q&A] [answered]
@devemux86 ist es angedacht in Zukunft das Headup Display von Tilsberk anzubinden? 
@devemux86 Are there plans to connect the Tilsberk head-up display in the future?

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider 

Thanks for the suggestion.
We may see this in the future as we are working on many other expected features.

Danke für den Vorschlag.
Wir können dies in Zukunft sehen, da wir an vielen anderen erwarteten Funktionen arbeiten.

**Comment by ToToGSARider** [comment]
@devemux86 

Wie lange könnte es dauern, bis so eine Anbindung umgesetzt ist? 

How long could it take to implement such a connection?

**Reply by devemux86** [reply]
@ToToGSARider 

We cannot provide estimates for the implementation of new functions.
This also depends on how popular a function is and how much work it requires.

Wir können keine Schätzungen für die Implementierung neuer Funktionen abgeben.
Das hängt auch davon ab, wie beliebt eine Funktion ist und wie viel Arbeit sie erfordert.
